plugin/CactbotEventSource/CactbotEventSource.cs

```diff
@@ -624,6 +624,17 @@ struct OverlayPreset : IOverlayPreset {
       });
     }
 
+    private void RegisterExternalPreset(string name, string url, int width, int height, bool locked)
+    {
+        Registry.RegisterOverlayPreset(new OverlayPreset
+        {
+            Name = $"Cactbot External {name}",
```

Comment:
```suggestion
            Name = $"{name}",
```

I think it'd be fine to just call this `ZeffUI`.  I think calling it a cactbot anything will likely be more confusing to users than just calling it what it is.  I think I'm just looking towards the future where there's not a cactbot plugin, and so this would be `ZeffUI` in a world where you could just add this to OverlayPlugin directly.

Reply:
Yeah that's fine. Make sure to let me know when OverlayPlugin is far enough along for me to switch over :)